webapp/README.md

```diff
@@ -92,15 +92,15 @@ You can then visit the Storybook playground on `http://localhost:3002`
 
 ## Styleguide Migration
 
-We are currently in the process of migrating our styleguide components and design tokens from the [Nitro Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) into the main [Human Connection repository](https://github.com/Human-Connection/Human-Connection) and refactoring our components in the process. During this migration, our new components will live in a `view` folder to separate them from the old, yet untouched components.
+We are currently in the process of migrating our styleguide components and design tokens from the [Nitro Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) into the main [Human Connection repository](https://github.com/Human-Connection/Human-Connection) and refactoring our components in the process. During this migration, our new components will live in a `_new` folder to separate them from the old, yet untouched components.
 
 ### Folder Structure
 
-The folder structure we are aiming for is based on the [directory setup proposed by Nuxt.js](https://nuxtjs.org/guide/directory-structure):
+The folder structure we are following is [prescribed by Nuxt.js](https://nuxtjs.org/guide/directory-structure):
 
-- **assets** contains icons, images and logos in `svg` format
-- **components** are the generic building blocks of the app – small, reusable and usually not coupled to state
-- **features** are composed of components but tied to a particular function of the app (e.g. `comment` or `post`)
+- **assets** contains icons, images and logos in `svg` format and all shared SCSS files such as `tokens`
+- **components** separated into two sub-folders:
+    - **generics** are the generic building blocks of the app – small, reusable and usually not coupled to state
+    - **features** are composed of components but tied to a particular function of the app (e.g. `comment` or `post`)
```

Comment:
I was wondering if we could change up the naming convention here? Or if you could talk about your motives for naming it so? I believe what you are trying to achieve here is to break these into `presentational` vs `container` components, or is there some nuance I'm not seeing?
I know I am familiar with this concept from `React`, but have also found some info about in relation to `Vue`
I just feel like it might be easier for other developers to understand the distinction if it's divided by terms they have a higher likelihood to know. What do you think?

Reply:
I am happy to change the names to whatever we as a team decide makes the most sense! The reason why I went with `generics` and `features` over `presentational` and `container` components is that I've only ever seen the latter ones used in combination.  E.g., a `CommentList` folder would host a `Container.vue` to fetch and transform the needed data and a `View.vue` responsible for how the data is displayed. (I am not a big fan of this pattern, though.)

If you have a different understanding of the two, please share!

(I would still like to merge this PR soon, so if it gets another approval before we make a decision on the names I'll merge it – we can easily rename the folders in a separate PR. 🙂)

Reply:
I'm happy with `generics/` and `features/`, or maybe even `generic/` singular,
to avoid name-clash with generics in Java (or pharmaceuticals). I could also
think of `use-cases/`.

`Features` is certainly better than `container`. You could also call it
`context` my all-time favourite meaningless term. :wink: